accounts-db/src/ancient_append_vecs.rs

```diff
@@ -577,6 +604,14 @@ impl AccountsDb {
             })
             .count()
             .saturating_sub(randoms as usize);
+        // ideal storage size is total alive bytes of ancient storages / half of max ancient slots
+        tuning.ideal_storage_size = NonZeroU64::new(
+            (infos.total_alive_bytes.0 / tuning.max_ancient_slots.max(1) as u64 * 2).max(5_000_000),
```

Comment:
i stuck a magic number here (5M). we need some constant or description for this.